ckanext/facetcollapse/templates/package/search.html

```diff
@@ -10,6 +10,10 @@
   <a class="close no-text hide-filters"><i class="fa fa-times-circle"></i><span class="text">close</span></a>
 </div>
 
-  {% resource 'facetcollapse/facetcollapse.js' %}
+  {% if h.ckan_version() > '2.9' %}
```

Comment:
Use something like [this](https://github.com/ckan/ckanext-harvest/commit/762ee144020d45d11fd2a883f5c5ee76d12ec707) or a custom helper, otherwise it will fail with `2.10`. I've been bitten by this in other extensions. If you got this from the wiki page it needs to be updated